pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -406,6 +406,24 @@ type MonitoringSpec struct {
 	// Interval determines prometheus scrape interval
 	// +optional
 	Interval *metav1.Duration `json:"interval,omitempty"`
+
+	// Ceph exporter configuration
+	// +optional
+	Exporter *CephExporterSpec `json:"exporter,omitempty"`
```

Comment:
please update the CR in the examples folder with an example(comment it)

Reply:
done